parameter_files/fates_params_default.cdl

```diff
@@ -1385,8 +1387,6 @@ data:
 
  fates_eca_plant_escalar = 1.25e-05 ;
 
- fates_fire_active_crown_fire = 0 ;
```

Comment:
After conversation with @jkshuman 
I need to revert back to the scalar version of `fates_fire_active_crown_fire` as an on/off switch that doesn't vary by pft. Changes required in fates_params_default.cdl, EDPftvarcon, EDParamsMod, and SFMainMod.